lib/surface/compiler.ex

```diff
@@ -317,6 +318,24 @@ defmodule Surface.Compiler do
      }}
   end
 
+  defp convert_node_to_ast(
+         :unless,
+         {_, attributes, children, node_meta},
+         compile_meta
+       ) do
+    meta = Helpers.to_meta(node_meta, compile_meta)
+    default = %AST.AttributeExpr{value: false, original: "", meta: node_meta}
```

Comment:
```suggestion
    default = %AST.AttributeExpr{value: false, original: "", meta: meta}
```

The ast structures should all reference an ast.meta struct for their meta prop